devtools/php-cs-fixer/run-php-cs-fixer.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```
No bash features used, so it is not required.